README.md

```diff
@@ -122,6 +122,22 @@ MemFree is equipped with powerful features that cater to various search and prod
 ```
 curl -fsSL https://bun.sh/install | bash
 ```
+#### Bun Not Found Error
+
+#### Windows
```

Comment:
https://bun.sh/docs/installation

did you test your commands?

.bashrc is not usually used on Windows because Windows does not use the bash shell by default.